ci-operator/jobs/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master-presubmits.yaml

```diff
@@ -1067,6 +1067,136 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-ipi,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Not sure we need it to run everytime, so maybe we could start by making this false?